### DISCUSSION TITLE
Tipo de usuario

### DISCUSSION THREAD

**Discussion by SaryNotfound** [Curcolor/COMMUNITY#9] [Q&A] [answered]
¿Los usuarios que se registren como donantes se quedarán como donantes o también pueden ser recolectores?

Pienso que no deberíamos limitar a los usuarios, pero también pienso en como se vería el perfil, pienso que se entendería como “hace sus donaciones a punta de los recursos de otras personas” entonces no sé qué colocar, ayuden.

**Comment by JUNIORRDSR** [comment]
> [!WARNING]
>Nuestro sistema no se encargaría de recaudaciones, sería mas de ayuda misma de las personas a otras como apoyo presencial o cosas que no nos involucren en el manejo de recursos de terceros por que se nos complicaría el manejo de estos

**Reply by SaryNotfound** [reply]
Me refiero a que pueden ser donantes o recolector de los fondos, el que necesita una donación, no que esto recaude ni que maneje los recursos.

**Reply by JUNIORRDSR** [reply]
> [!NOTE]
> Entonces eso sería mas como una **Categoría** en su descripción, ya una vez creado el tipo de usuario no se podría cambiar

**Comment by Curcolor** [comment, marked as answer]
> [!TIP]
>Okey ya te entendí, mira doy una idea por un lado si quieres obtener donaciones o ayuda se podría hacer que tienes que entrar en esa cuenta y agregarte roles para que te aparezcan las opciones de esos otros tipos de usuarios .
